plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/actions/datasource/ToolsContextMenuHandler.java

```diff
@@ -78,7 +79,8 @@ public Object execute(ExecutionEvent event) throws ExecutionException
             }
         }
         menuManager.add(new Separator());
-        {
+        
+        if (part.getSite().getService(ICommandService.class).getCommand(SQLEditorCommands.CMD_SQL_EDITOR_OPEN).isEnabled()) {
```

Comment:
Use `ActionUtils#isCommandEnabled` instead. It looks prettier and it makes sure that the given command exists.